src/types/index.ts

```diff
@@ -0,0 +1,13 @@
+export type Object = {
+  [key: string]: any;
+};
```

Comment:
What is this for?

Reply:
I create Object as a standalone type as I thought I would reuse it in other places. And I think it works as the utility type you've mentioned. Thanks again for showing me the Record type!